deepchecks/nlp/text_data.py

```diff
@@ -91,13 +92,14 @@ class TextData:
     categorical_properties : t.Optional[t.List[str]] , default: None
         The names of the categorical properties columns. If None, categorical properties columns are automatically
         inferred. Only relevant if properties is not None.
-    embeddings : t.Optional[Union[pd.DataFrame, np.ndarray, str]] , default: None
-        The text embeddings for the samples. Embeddings must be given as either a pandas DataFrame or a path to a pandas
-        DataFrame compatible csv file, with the rows representing each sample and columns representing the different
-        embeddings dimensions. If None, no embeddings are set.
-        The number of rows in the embeddings DataFrame must be equal to the number of samples in the dataset, and the
-        order of the rows must be the same as the order of the samples in the dataset.
-        In order to calculate the default embeddings, use the `TextData.calculate_default_embeddings` function after
+    embeddings : t.Optional[Union[np.ndarray, pd.DataFrame, str]], default: None
```

Comment:
is pandas also an option or not any more?

Reply:
I thought about keeping it as a "secret" option. Not crashing the code, but not documented to keep it simple. WDYT?

Reply:
mm why not mentioned that also a pandas df will be OK? it's not a lot of extra text

Reply:
mmm ok nvm